roles/sap_general_preconfigure/tasks/sapnote/2369910.yml

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+- name: Configure - Display SAP note number 2369910 and its version
+  ansible.builtin.debug:
+    msg: "SAP note {{ (__sap_general_preconfigure_sapnotes_versions | selectattr('number', 'match', '^2369910$') | first).number }}
+          (version {{ (__sap_general_preconfigure_sapnotes_versions | selectattr('number', 'match', '^2369910$') | first).version }}): SAP Software on Linux: General Information"
+  tags:
+    - always
+
+- name: Check locales
+  when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2369910 | d(false)
+  tags:
+    - sap_general_preconfigure_2369910
+    - sap_general_preconfigure_configure_locale
+  block:
+    - name: Get list of installed locales
+      ansible.builtin.command: locale -a
+      changed_when: false
+      register: __sap_general_preconfigure_locales_installed
+
+    - name: Assert that an English locale is installed
+      ansible.builtin.assert:
+        that: __sap_general_preconfigure_locales_installed.stdout_lines | select('match', '^en_') | list | length > 0
+        fail_msg: "FAIL: No English locale is installed. Please install an English locale!"
+        success_msg: "PASS: An English locale is installed."
+
+    - name: Get the current default locale
+      ansible.builtin.command: awk '/^LANG=/&&/en_/{print}' /etc/locale.conf
```

Comment:
this does not take into account that C.UTF-8 is also english

Reply:
You are right. Although C is English it is not sufficient for HANA2 SPS08. The en_* locales have to be installed. 
A test run showed that having the system locale C while having en_* installed was sufficient.
I am not sure how we can deal with it. For now I suggest accepting C and en_* locales, as long as en_* is installed. IIRC SAP recommends en_US.UTF-8, so if some body has a proof for that and put here or to the issue would be great.

Reply:
suggested new `awk` statement:
`awk '/^LANG=/&&(/C.UTF-8/||/en_/){print}' /etc/locale.conf`